psutil/_psbsd.py

```diff
@@ -894,8 +894,10 @@ def _not_implemented(self):
     if HAS_PROC_OPEN_FILES:
 
         @wrap_exceptions
-        def open_files(self):
+        def open_files(self, only_regular=True):
             """Return files opened by process as a list of namedtuples."""
+            # <<< TODO >>>: the regular file filtering is done in the
+            # underlying c code: `psutil/arch/bsd/proc.c`
```

Comment:
I am pretty sure I can make this change. I just don't want to take the time to do it until I've gotten some indication from a maintainer that y'all would be willing to merge this PR.